Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/SpatialLocalization/HostCoordinateLocalizationMechanism.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Experimental.SpatialAlignment.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    /// <summary>
+    /// Helper localization method here the host finds a QR code and gives it to the observer
+    /// </summary>
+    internal abstract class HostCoordinateLocalizationMechanism : LocalizationMechanismBase
+    {
+        private readonly object lockObject = new object();
+        private readonly List<Guid> guids = new List<Guid>();
```

Comment:
>private readonly List<Guid> guids = new List<Guid>(); [](start = 8, length = 53)

I don't see this list actually used anywhere, is it necessary to store the list of Guids?

Reply:
Not for this case.